content/300-guides/200-deployment/110-deployment-guides/650-caveats-when-deploying-to-aws-platforms.mdx

```diff
@@ -44,9 +44,9 @@ This error happens because AWS Elastic Beanstalk doesn't install `devDependencie
 AWS Lambda defines an **deployment package upload limit**, which includes:
 
 - All application code
-- Binaries like the [Prisma query engine](../../../concepts/components/prisma-engines/query-engine)
+- Binaries like the [Prisma query engine](/concepts/components/prisma-engines/query-engine)
 
-The [deployment package (.zip) size limit for lambdas is 50MB](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html). When you prepare a deployment package, remove any files that the function does not require in production to keep the final .zip as small as possible. This includes some [Prisma engine binaries](#deleting-prisma-engine-binaries-that-are-not-required).
+The [deployment package (.zip) size limit for lambdas is 50MB](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html). When you prepare a deployment package, remove any files that the function does not require in production to keep the final .zip as small as possible. This includes some [Prisma engine binaries](/concepts/components/prisma-engines).
```

Comment:
Hm, until now that was an internal link to the section about how to get rid of some engines. Now it is a "further information about engines over here" link. Intentional and wanted?

Reply:
My bad